docs/src/main/sphinx/installation/cli.rst

```diff
@@ -37,6 +37,27 @@ password value to avoid the prompt.
 Use ``--help`` to see information about specifying the keystore, truststore, and
 other authentication details as required. If using Kerberos, see :doc:`/security/cli`.
 
+External authentication - SSO
+-----------------------------
+
+The  ``--external-authentication`` option is used for browser-based SSO
+authentication, as detailed in :doc:`/security/oauth2`. With this configuration,
+the CLI displays an URI that you have to open in a web browser for authentication.
+Due to the above, this can only be used when you have an access to a web-browser,
+so that you can copy-paste the displayed uri.
+
+The expected behaviour is as follows:
+ - user opens up CLI with ``--external-authentication`` option enabled,
+ - CLI opens up and allows a user to start writing queries,
+ - on a first query, a request to open up a browser will be displayed,
+ - after completing the login, user's query will be executed and its results displayed,
```

Comment:
* Open the URL in a browser and follow through the authentication process
* The CLI automatically receives a token from the authentication process
* Successful login in the browser, allows the CLI to proceed with executing the query ba